requests/packages/urllib3/connection.py

```diff
@@ -6,18 +8,13 @@
 from .packages import six
 
 try:  # Python 3
-    from http.client import HTTPConnection as _HTTPConnection, HTTPException
+    from http.client import HTTPConnection as _HTTPConnection
```

Comment:
Specifically I'm talking about:

- https://github.com/timothycrosley/pies/issues/49
- https://github.com/PythonCharmers/python-future

They both cause an incredible amount of breakage for this pattern of Python 2/3 compat (that doesn't happen when using `six.moves`).